build/crd/kustomization.yaml

```diff
@@ -10,10 +10,16 @@ patchesJson6902:
     version: v1
     kind: CustomResourceDefinition
     name: postgresclusters.postgres-operator.crunchydata.com
-  path: validation.yaml
+  path: condition.yaml
 - target:
     group: apiextensions.k8s.io
     version: v1
     kind: CustomResourceDefinition
     name: postgresclusters.postgres-operator.crunchydata.com
   path: todos.yaml
+- target:
+    group: apiextensions.k8s.io
+    version: v1
+    kind: CustomResourceDefinition
+    name: postgresclusters.postgres-operator.crunchydata.com
+  path: validation.yaml
```

Comment:
What happens if we mix up the order of these patches? Do any of them touch the same items?

Reply:
They might and it could go badly. This is one of the reasons we store the final result in Git and approve the diffs in PRs.